### ISSUE TITLE
NVDA does not read showInputBox content correctly

### ISSUE THREAD

**Issue by cait1inhennessy** [Microsoft/vscode#58105]
Repro Steps:
1. Download Live Share extension for VSCode
2. Go to the "share" icon in the status bar, click it, and sign in to start a collaboration session
3. Click the name icon in the status bar
![capture4](https://user-images.githubusercontent.com/16658549/45174888-d29ad600-b1c0-11e8-8c28-b5ed47a16182.PNG)
4. Select "share server" in the quick pick menu that appears
5. Type any number in "Enter port to expose..."
6. Verify that NVDA reads the user input and the prompt, with no extra information

OR
1. Show any input box with vscode.window.showInputBox API, and provide a non-null value for the prompt option
2. Enter some input in the input box
3. Verify that NVDA reads the user input and the prompt, with no extra information

Actual Result:  
NVDA reads some background content before reading the input entered by the user, **and** NVDA does not read the prompt under the input box

Expected Result: 
NVDA reads the input entered by the user and the prompt under the input box. NVDA does not read any extra information.

User Impact: 
Screen Reader user will find difficulty in using the application if it does not read the contents and controls that are appearing on the screen properly

MAS Reference: https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D 

OS: Windows 10 Enterprise 17134.228

**Comment by cait1inhennessy** [comment]
![capture5](https://user-images.githubusercontent.com/16658549/45175639-c152c900-b1c2-11e8-84d1-8299f6936b45.PNG)
Here is our input box ^

**Comment by cait1inhennessy** [comment]
Possible dupe of https://github.com/Microsoft/vscode/issues/55186
